src/pages/Deeplink.vue

```diff
@@ -0,0 +1,212 @@
+              name="fname"
+              value=""
+              class="px-6 py-3 rounded-3xl"
+            /><br />
+            <div class="py-2" />
+            <label
+              for="fname"
+              class="text-xl font-bold text-true-gray-800 dark:text-gray-100"
+              >Generated Link</label
+            ><br />
+            <label
+              id="flink"
+              class="text-xl font-bold text-true-gray-800 dark:text-gray-100"
+            ></label
+            ><br />
+            <div class="py-2" />
+            <a
+              v-on:click="CheckUrl"
+              class="px-12 py-3 rounded-3xl font-medium bg-gradient-to-b from-red-600 to-red-700 hover:from-red-900 text-white outline-none focus:outline-none hover:shadow-md hover:from-true-gray-900 transition duration-200 ease-in-out"
+              >CheckUrl</a
+            >
+            <div class="py-4" />
+            <!-- </div> -->
+          </div>
+          <!-- /hero section -->
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters, mapState } from "vuex";
+import queryString from "@/utils/queryString";
+import config from "@/config";
+
+export default {
+  name: "Landing",
+
+  computed: {
+    ...mapGetters({ auth: "ifAuthenticated" }),
+    ...mapState(["user", "stateParam"]),
+    state() {
+      return btoa(this.stateParam);
+    },
+    username() {
+      return this.user
+        ? `${this.user.username}#${this.user.discriminator}`
+        : "Anonymous";
+    },
+    loginUrl() {
+      const loginParams = {
+        client_id: config.clientId,
+        redirect_uri: config.home,
+        response_type: "token",
+        scope: "identify guilds",
+        state: this.state,
+      };
+      return `${config.discordApi}/oauth2/authorize${queryString(loginParams)}`;
+    },
+  },
+  data() {
+    return {
+      botUrl: `https://discord.com/api/oauth2/authorize?client_id=${config.clientId}&permissions=268438560&scope=bot`,
+      coins: [],
+    };
+  },
+  methods: {
+    CheckUrl() {
+      var CoinName, CurrencyType, Ammount, Memo;
+      CoinName = document.getElementById("FCoinName").value;
+      CurrencyType = ""
+      if( document.getElementById("COIN").checked ) {
+        CurrencyType = "COIN"
+      } else if (document.getElementById("USD").checked){
+        CurrencyType = "USD"
+      }
+      Ammount = document.getElementById("FAmmount").value;
+      Memo = document.getElementById("FMemo").value;
+      if (CoinName || CurrencyType || Ammount || Memo) {
+        var url =
+          "https://www.rally.io/creator/" +
+          CoinName +
+          "?inputType=" +
+          CurrencyType +
+          "&amount=" +
+          Ammount +
+          "note=" +
+          Memo;
+        document.getElementById("flink").innerHTML = url;
+      } else {
+        document.getElementById("flink").innerHTML = "Error! fill all filds";
+      }
+      return url;
+    },
+    fetchCoins() {
```

Comment:
No need for `fetchCoins` dashboard does this when it's mounted so you can get that from the state manager

```js
computed: {
    ...mapState(["coins"]),
},
```